server/app.js

```diff
@@ -74,7 +75,28 @@ if (!process.env.SB_FILE_STORE) {
   throw new Error('SB_FILE_STORE must be specified')
 }
 const fileStoreSettings = JSON.parse(process.env.SB_FILE_STORE)
-setStore(new LocalFileStore(fileStoreSettings.filesystem))
+if (!(fileStoreSettings.filesystem || fileStoreSettings.doSpaces)) {
```

Comment:
nit: I'd probably just check that this is truthy here, and throw a different error in an `else` clause below (if no valid key could be found in it). That will make it slightly easier to add/change the keys in the future (even though that's probably rare :) )